libs/sdk-ui-dashboard/src/insight/drillingUtils.ts

```diff
@@ -0,0 +1,77 @@
+// (C) 2020-2021 GoodData Corporation
```

Comment:
contemplating whether these utils are actually part of some model that we want to have in some other package. e.g somewhere with the rest of the drill logic?

perhaps this should eventually be in `sdk-ui` package? if so, let's move the stuff under _staging so that we can redistribute it later.